src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -16,9 +13,15 @@
 public class User {
 
     @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
+    @GeneratedValue
     private Long id;
     private String username;
     private String status;
     private Long userKey;
```

Comment:
Wiem że UserDto ktoś już wrzucił  ale pytanie czy 
userKey to pole które na pewno chcemy mapować do DTO?  Czy te pole nie powinno być tajne?